openreview/venue_request/venue_request.py

```diff
@@ -199,6 +199,107 @@ def setup_review_stage(self):
+                'value-radio': [
+                    'Yes, ethics reviews should be revealed publicly when they are posted',
+                    'No, ethics reviews should NOT be revealed publicly when they are posted'
+                ],
+                'required': True,
+                'default': 'Yes, ethics reviews should be revealed publicly when they are posted',
+                'order': 3
+            },
+            'release_ethics_reviews_to_authors': {
+                'description': 'Should the ethics reviews be visible to paper\'s authors immediately upon posting? Default is "No, ethics reviews should NOT be revealed when they are posted to the paper\'s authors".',
+                'value-radio': [
+                    'Yes, ethics reviews should be revealed when they are posted to the paper\'s authors',
+                    'No, ethics reviews should NOT be revealed when they are posted to the paper\'s authors'
+                ],
+                'required': True,
+                'default': 'No, ethics reviews should NOT be revealed when they are posted to the paper\'s authors',
+                'order': 4
+            },
+            'release_ethics_reviews_to_reviewers': {
+                'description': 'Should the reviews be visible to all reviewers, all assigned reviewers, assigned reviewers who have already submitted their own review or only the author of the review immediately upon posting?',
+                'value-radio': [
+                    'Ethics reviews should be immediately revealed to all reviewers and ethics reviewers',
+                    'Ethics reviews should be immediately revealed to the paper\'s reviewers and ethics reviewers',
+                    'Ethics reviews should be immediately revealed to the paper\'s ethics reviewers',
+                    'Ethics Review should not be revealed to any reviewer, except to the author of the ethics review'
+                ],
+                'required': True,
+                'default': 'Review should not be revealed to any reviewer, except to the author of the review',
+                'order': 5
+            },
+            'ethics_review_submissions': {
+                'order' : 6,
+                'value-regex': '.*',
+                'required': True,
+                'description': 'Comma separated values of submission numbers that need ethics reviews.'
+            },
+            'additional_ethics_review_form_options': {
+                'order' : 7,
+                'value-dict': {},
+                'required': False,
+                'description': 'Configure additional options in the ethics review form. Use lowercase for the field names and underscores to represent spaces. The UI will auto-format the names, for example: supplementary_material -> Supplementary Material. Valid JSON expected.'
+            },
+            'remove_ethics_review_form_options': {
```

Comment:
This appears as a big text box instead of the usual "Remove __ Options" size

Reply:
that's weird and I'm using the same field definition from other stages